java/openmldb-jdbc/src/main/java/com/_4paradigm/openmldb/sdk/Common.java

```diff
@@ -48,7 +49,32 @@ public static int type2SqlType(DataType dataType) throws SQLException {
         }
     }
 
-    public static com._4paradigm.openmldb.sdk.Schema convertSchema(com._4paradigm.openmldb.Schema schema) throws SQLException {
+    public static DataType toDataType(int sqlType) throws SQLException {
```

Comment:
`public static DataType swigToEnum(int swigValue)` can do convert int to DataType